cmake/legateboost/add_cffi.cmake

```diff
@@ -0,0 +1,116 @@
+#=============================================================================
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-NvidiaProprietary
+#
+# NVIDIA CORPORATION, its affiliates and licensors retain all intellectual
+# property and proprietary rights in and to this material, related
+# documentation and any modifications thereto. Any use, reproduction,
+# disclosure or distribution of this material and related documentation
+# without an express license agreement from NVIDIA CORPORATION or
+# its affiliates is strictly prohibited.
+#=============================================================================
+
+include_guard(GLOBAL)
+
+function(legateboost_add_cffi py_path)
```

Comment:
Does it make more sense to just commit a python file instead of a cmake file generating a python file? Or do we need something from cmake?

Reply:
🤔 you raise an excellent point, let me double check this

Reply:
Yeah, can just check in `library.py` directly, did so